Demo/Sources/Views/LongPressPlaybackView.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import PillarboxPlayer
+import SwiftUI
+
+#if os(iOS)
+private struct LongPressView<Content>: View where Content: View {
+    @GestureState private var isLongPressing = false
+    @State private var timer: Timer?
+
+    let minimumPressDuration: TimeInterval
+    @ViewBuilder let content: () -> Content
+    let action: (_ finished: Bool) -> Void
+
+    var body: some View {
+        content()
+            .simultaneousGesture(longPressGesture())
+            .onChange(of: isLongPressing) { isChanged in
+                timer?.invalidate()
+                if !isChanged {
+                    action(true)
+                } else {
+                    timer = Timer.scheduledTimer(withTimeInterval: minimumPressDuration, repeats: false) { _ in
+                        action(false)
+                    }
+                }
+            }
+    }
+
+    init(minimumPressDuration: TimeInterval = 1, content: @escaping () -> Content, action: @escaping (_ finished: Bool) -> Void) {
+        self.minimumPressDuration = minimumPressDuration
+        self.content = content
+        self.action = action
+    }
+
+    private func longPressGesture() -> some Gesture {
+        LongPressGesture(minimumDuration: .infinity, maximumDistance: .infinity)
+            .updating($isLongPressing) { value, state, transaction in
+                state = value
+                transaction.animation = .easeIn(duration: 2.0)
+            }
+    }
+}
+
+private struct LongPressPlaybackInfoView: View {
+    var body: some View {
+        HStack {
+            Text("x 2")
+                .font(.footnote)
+                .bold()
+            HStack(spacing: 3) {
+                ForwardArrowView()
+                ForwardArrowView()
+            }
+        }
+        .padding(10)
+        .foregroundStyle(.white)
+        .background(.black.opacity(0.7))
+        .clipShape(RoundedRectangle(cornerRadius: .infinity))
+    }
+}
+
+private struct ForwardArrowView: View {
+    var body: some View {
+        Image(systemName: "arrowtriangle.forward.fill")
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+            .frame(width: 10)
+    }
+}
+
+struct LongPressPlaybackView<Content>: View where Content: View {
+    @State private var speed: Float?
+    @State private var isFinished = true
+    @State private var isBusy = false
```

Comment:
Is busy state observation really essential for correct behavior?

Reply:
If we want to avoid displaying the bubble when the player cannot speed up, then yes. I find it strange to show the bubble to the user when speeding up does nothing.